examples/esp32-wroom-32/test_ds18b20/main.c

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Eduardo Azócar <eduardo@turpialdev.com>
```

Comment:
Wrong  copyright formar

Reply:
This is the same format in the coding convention (license and header copyright)